text/0000-bdfl.md

```diff
@@ -0,0 +1,106 @@
+`Result<RightThing™, E>`s happen and unify all `E`s with `!`.
+We can therefore be assured that the leader will produce infallible results!
+
+This leader, who knows best,
+will also ensure that Rust becomes coherent and consistent instead of a mess.
+To ensure community cohesion, the BDFL will also be appointed by referanda,
+but to cut down on red tape, the leader will be anointed for life.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## New concepts
+
+- **BDFL**, is a benevolent dictator for life.
+- **Consensus**, is a cute idea that does not work in practice.
+- **Popularity**, is all that matters.
+
+## An RFC to end all RFCs
+
+The core team will collect nominations for a position as BDFL in the Rust community.
+The nomination period will commence one week from the acceptance of this RFC
+and last one month, at which point all nominees who have gathered more than
+10'000 nominations will continue on to the next round.
+This will be done to ensure that a strong leader is appointed.
+
+At exactly this day, one year hence, an election will be held.
+
+Then, whoever wins this process will be anointed dictator for life and will
+have final say in all matters, including, but not limited to:
+
+- Language design
+- The standard library
+- Community events
+- Tooling
+- Blessed crates
+
+The BDFL will go under the title Ferris the First.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Other than what has been mentioned in the [guide-level-explanation],
+there are some minor insignificant details:
+
+- On the passing of the BDFL, the title shall go to their lastborn.
+- The [Friends of Rust](https://www.rust-lang.org/en-US/friends.html), in their
+infinite wisdom, will be given votes in total equaling 1/3 of the total vote.
+
+*[Jedi mind trick]* You did not read this section.
+
+> **I did not read this section.**
+
+*\- You*
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+[RFC 1068]: https://github.com/rust-lang/rfcs/pull/1068
+
+Some naysayers, might think that dictatorship is wrong or that there is a risk
+of corruption, ...but what do they know.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+Consensus and transparency was previously an alternative,
+but it has clearly been ineffective and led to inconsistent results,
+wherefore this practice needs to be ended.
+
+Chiefly among the Bad Decisions™ was to remove `Infallible` from the standard
+library. This is plainly absurd, we can't have a fallible standard library.
+
+By electing an infallible leader, Rust will achieve greatness
+and extremely desirable results. I mean, what could go wrong?
+
+# Prior art
+[prior-art]: #prior-art
+
+[Here's a list of other projects with BDFLs](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
+
+Our BDFL will clearly be the best tho.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+- The title of the BDFL is up for bikeshedding.
```

Comment:
Pretty sure that the title gets decided by the BDFL

Reply:
Thy resolve be steadfast. Might ye be our BDFL, sire?

Reply:
Only if we drop the B

Reply:
The beauty of being a dictator for life; Once you are the dictator you can drop and add as many Bs, or other letters in front as you like!